static/app/views/performance/cache/settings.ts

```diff
@@ -19,7 +19,7 @@ export const CHART_HEIGHT = 160;
 export const CACHE_BASE_URL = `/performance/${BASE_URL}`;
 
 export const BASE_FILTERS: SpanMetricsQueryFilters = {
-  'span.op': 'cache.get_item', //  TODO - add more span ops as they become available, we can't use span.module because db.redis is also `cache`
+  'span.op': '[cache.get_item,cache.get]', //  TODO - add more span ops as they become available, we can't use span.module because db.redis is also `cache`
```

Comment:
😭 more motivation to work on a `MutableQuery` alternative that handles array conditions properly. Can you add a second `TODO` about how it's awkward to have to construct an array string here, and why that is?

Reply:
👍 good idea, i'll add another todo.